types/ethers-contracts/factories/Counter__factory.ts

```diff
@@ -0,0 +1,118 @@
+/* Autogenerated file. Do not edit manually. */
+/* tslint:disable */
+/* eslint-disable */
+import type { Addressable } from "ethers";
+import {
+    Contract,
+    ContractFactory,
+    ContractTransactionResponse,
+    Interface
+} from "ethers";
+import type { Signer, ContractDeployTransaction, ContractRunner } from "ethers";
+import type { NonPayableOverrides } from "../common.js";
+import type { Counter, CounterInterface } from "../Counter.js";
```

Comment:
@daiagi are we generating these files? Not sure where this is used and where it's coming from

Reply:
no idea where they came from, removed